middleware/common/source/service/lookup.cpp

```diff
@@ -69,7 +69,6 @@ namespace casual
             swap( lhs.m_correlation, rhs.m_correlation);
          }
 
-
          void Lookup::update( Reply&& reply)
```

Comment:
I decided against removing this. Resetting m_correlation is used to indicate that we dont need to discard the pending lookup so it felt a bit tricky breaking this out into a free function.